packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -585,7 +642,8 @@ export class TogglingHelperConditionalsTest extends TogglingConditionalsTest {
     this.assertText('T1F2');
   }
 
-  ['@test it tests for `isTruthy` on the context if available']() {
+  // This no longer makes sense for glimmer since `this` refers to the component, which cannot be a proxy
+  ['@htmlbars it tests for `isTruthy` on the context if available']() {
```

Comment:
The ArrayTestCases and ObjectTestCases do provide coverage for proxy content changing, and are mixed into the `TogglingConditionalsTest`.

`with-tests.js` extends `TogglingSyntaxConditionalsTest` which in turn is an extension of `TogglingConditionalsTest`. `each-tests.js` explicitly mixes in the ArrayTestCases so i'd say we're covered